src/builder.ts

```diff
@@ -52,10 +52,16 @@ export default (defaults: Partial<OrkaOptions> = _defaults) => {
   diamorphosis(config, options);
 
   options.appName ||= config?.app?.name;
-  if (require.cache[require.resolve('koa')]) logger.warn('Koa was initialized before orka');
-  if (config.mongodb?.url && require.cache[require.resolve('mongoose')]) logger.warn('Mongoose was initialized before orka');
-  if (config.queue?.url && require.cache[require.resolve('amqplib')]) logger.warn('Amqplib was initialized before orka');
-  if (config.postgres?.url && require.cache[require.resolve('pg')]) logger.warn('postgres was initialized before orka');
```

Comment:
❓ Don't we also need the condition for the config part? Why it can be omitted?

Reply:
The condition was needed because pg for example might not be installed and require would throw an error. The error is caught in the new implementation